test/step-definitions/responses.ts

```diff
@@ -0,0 +1,49 @@
+import { binding, then, when, before} from 'cucumber-tsflow';
+import { assert, expect } from 'chai';
+import { Test, TestingModule } from '@nestjs/testing';
+import * as request from 'supertest';
+import { AppModule } from '../../src/app.module';
+
+class Context {
+  public app;
+  public response;
+}
+
+// tslint:disable-next-line:max-classes-per-file
+@binding([Context])
+export class HelloWorldSteps {
+  constructor(protected context: Context) {}
+
+  @before()
+  public async before(): Promise<void> {
+    const moduleFixture: TestingModule = await Test.createTestingModule({
+      imports: [AppModule],
+    }).compile();
+
+    this.context.app = moduleFixture.createNestApplication();
+    await this.context.app.init();
+  }
+
+  @when(/make a GET Request to "([^"]*)"/)
+  public async callToGETAPI(url: string) {
```

Comment:
`GETAPI` this in capitals makes it difficult to read.

What about the method name to be something like `getRequest`?